src/app/layout.tsx

```diff
@@ -22,31 +23,35 @@ export function generateMetadata(): Metadata {
 }
 export default function RootLayout({ children }: PropsWithChildren) {
   return (
-    <html lang="en" className="h-full">
+    <html lang="en" className="h-full" suppressHydrationWarning>
```

Comment:
I don't think we should add this. We should investigate and resolve warnings instead of suppressing them.